MAUI/CheckBox/Events.md

```diff
@@ -55,4 +55,38 @@ This event occurs when the value or state of the [`IsChecked`](https://help.sync
 {% endhighlight %}
 {% endtabs %}
 
-![.NET MAUI CheckBox](Images/Getting-Started/tristate.png)
\ No newline at end of file
+![.NET MAUI CheckBox](Images/Getting-Started/tristate.png)
+
+## StateChanging event
+
+The `StateChanging` event is triggered when the state of the [IsChecked](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html#Syncfusion_Maui_Buttons_SfCheckBox_IsChecked) property is about to change, either by tapping the CheckBox control or by programmatically setting the IsChecked value using XAML or C# code. The event arguments are of type `StateChangingEventArgs` and provide the following properties:
```

Comment:
This event only triggers when change the state by tapping.

Reply:
Updated the content.